Uppsala/UppEt41.xml

```diff
@@ -0,0 +1,501 @@
+                                        
+                                        <item xml:id="q11" n="10">
+                                            <num value="10">፲</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="76r" to="83v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q12" n="11">
+                                            <num value="11">፲፩</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="84r" to="91v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q13" n="12">
+                                            <num value="12">፲፪</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="92r" to="99v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q14" n="13">
+                                            <num value="13">፲፫</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="100r" to="107v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q15" n="14">
+                                            <num value="14">፲፬</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="108r" to="115v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q16" n="15">
+                                            <num value="15">፲፭</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="116r" to="123v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q17" n="16">
+                                            <num value="16">፲፮፡</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="124r" to="131v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q18" n="17">
+                                            <num value="17">፲፯</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="132v" to="139v"></locus>
```

Comment:
The quire normally starts with 132r. If 132v is part of the quire, then 132r must also be part of this quire. You should either remove all r and v or you change 132v to 132r.

Reply:
Well spotted!